go.mod

```diff
@@ -34,7 +34,7 @@ require (
 	github.com/rivo/uniseg v0.2.0 // indirect
 	github.com/spf13/cobra v1.2.0
 	github.com/submariner-io/admiral v0.12.0-m0
-	github.com/submariner-io/cloud-prepare v0.12.0-m0
+	github.com/submariner-io/cloud-prepare v0.12.0-m0.0.20211115085640-9f6a4bc47cf9
```

Comment:
We should update this to include https://github.com/submariner-io/cloud-prepare/pull/143 which just got merged - it fixes a significant bug.